pkg/kubernetes/topology.js

```diff
@@ -57,8 +57,22 @@ define([
                 var all = client.select();
                 client.track(all);
                 $(all).on("changed", digest);
+                $(all).on("updated", entity_updated);
+                $(all).on("removed", entity_removed);
                 digest();
 
+                function entity_updated(ev, entity, key, last) {
+                    if($scope.selected.kind === last.kind && $scope.selected.key === last.key) {
+                        $scope.selected = last;
```

Comment:
Here you're setting the selected to the old entity. I don't think you want to do that.